### ISSUE TITLE
Feature: Dungeon as siege

### ISSUE THREAD

**Issue by poaq** [suriyun-production/mmorpg-kit-docs#241]
Once in the week dungeon can be contested in gvg. 

In the week guild can apply by npc to contest dungeon - 3 guilds wich pledge the highest bid parctipate in the fight. 

Once guild aquire dungeon it gets access to rules modification of the dungeon and area that belongs to zone. (example: forced pvp, no respawns around dungeon, no teleports)
Every week guild that own dungeon get 10% of all gold that droped in dungeon - gm get it by npc and can share it with rest of guild. 
Every week guild has to defend dungeon. 

4 respawns in each dungeon. Main objective is to first capture single checkpoints and then destory main central crystal. Battle time: 60 min.

1. Guild can forms alliance (8 party x 8 ppl) / temporary access to ally chat.

**Comment by 1isenok** [comment]
yes, there aren’t enough such events, I would suggest you release them as a separate add-on package for 3D MMORPG KIT, I think not only I would buy such packages alone for extra money
